osutil/mountentry_linux_test.go

```diff
@@ -218,6 +216,30 @@ func (s *entrySuite) TestParseMountEntry6(c *C) {
 	c.Assert(e.CheckPassNumber, Equals, 7)
 }
 
+func (s *entrySuite) TestParseMountEntrySpacesInOptions(c *C) {
+	const p9DockerSpaces = `C:\134Program\040Files\134Docker\134Docker\134resources /Docker/host 9p rw,dirsync,noatime,aname=drvfs;path=C:\Program Files\Docker\Docker\resources;symlinkroot=/mnt/,mmap,access=client,msize=65536,trans=fd,rfd=4,wfd=4 0 0`
+	e, err := osutil.ParseMountEntry(p9DockerSpaces)
+	c.Assert(err, IsNil)
+
+	c.Assert(e.Name, Equals, "C:\\Program Files\\Docker\\Docker\\resources")
+	c.Assert(e.Dir, Equals, "/Docker/host")
+	c.Assert(e.Type, Equals, "9p")
+	c.Assert(e.Options, DeepEquals, []string{
+		"rw",
+		"dirsync",
+		"noatime",
+		"aname=drvfs;path=C:\\Program Files\\Docker\\Docker\\resources;symlinkroot=/mnt/",
```

Comment:
it'd be nice to figure out where this is coming from and maybe fix the kernel side as well, looking at https://elixir.bootlin.com/linux/v6.10/source/fs/9p/v9fs.c#L116 the kernel bits may need some escaping

Reply:
I've prepared a kernel patch and I'll send it once I figure out the right mailing list.